src/multi-http-loader.ts

```diff
@@ -13,14 +13,14 @@ export interface ITranslationResource {
 
 export class MultiTranslateHttpLoader implements TranslateLoader {
   constructor(
-    private http: HttpClient,
+    private handler: HttpBackend,
```

Comment:
HttpBackend needs to be imported at the top of the file.

Reply:
Done